lib/openhab/log/logger.rb

```diff
@@ -18,6 +18,11 @@ class Logger
       # @return [Array] Supported logging levels
       LEVELS = %i[TRACE DEBUG WARN INFO ERROR].freeze
```

Comment:
It seems that we could just use lowercase here, and avoid having to call `to_s` everywhere. LEVELS can be made private so as not to pollute the global namespace if necessary.

Reply:
This wasn't written by me, but can make that change as well, it doesn't seem to be used outside the Logger module.

Reply:
I understand it was written by @boc-tothefuture originally. It is something I was going to suggest changing, so why not just do it now :) WDYT @boc-tothefuture ?

Reply:
I missed this ping.. yep.. no problem, lets downcase and make it a private_constant